internal/service/studies/assets.go

```diff
@@ -0,0 +1,191 @@
+package studies
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"slices"
+
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
+	openapi "github.com/ucl-arc-tre/portal/internal/openapi/web"
+	"github.com/ucl-arc-tre/portal/internal/types"
+)
+
+func (s *Service) validateAssetData(assetData openapi.AssetBase) (*openapi.AssetCreateValidationError, error) {
+	// Validate title
+	if strings.TrimSpace(assetData.Title) == "" {
```

Comment:
I'd be tempted to define regex patterns for all of these fields and do away with the `TrimSpace` etc.